src/lib.rs

```diff
@@ -81,7 +81,7 @@
 //! #![feature(proc_macro, specialization)]
 //!
 //! extern crate pyo3;
-//! use pyo3::{py, Python, PyResult, PyModule, PyString};
+//! use pyo3::{py, Python, PyResult, PyModule, PyString, ObjectProtocol};
```

Comment:
Hmm, I don't think we should ask end user to import `ObjectProtocol` when they don't need it? #43 #44